internal/app/machined/pkg/controllers/network/route_spec.go

```diff
@@ -120,7 +120,7 @@ func (ctrl *RouteSpecController) Run(ctx context.Context, r controller.Runtime,
 	}
 }
 
-func findRoutes(routes []rtnetlink.RouteMessage, family nethelpers.Family, destination netip.Prefix, gateway netip.Addr, table nethelpers.RoutingTable) []*rtnetlink.RouteMessage {
+func findRoutes(routes []rtnetlink.RouteMessage, family nethelpers.Family, destination netip.Prefix, gateway netip.Addr, table nethelpers.RoutingTable, priority uint32) []*rtnetlink.RouteMessage {
```

Comment:
Maybe we can pass `RouteSpecSpec` directly? 6 arguments looks a bit too much.

Reply:
yep, thanks, updated